make_locale.py

```diff
@@ -19,11 +19,11 @@ def get_control_center_translations(lang):
     with open(infile, 'r') as f:
         fdata = f.read()
 
-    pattern = 'msgctxt "proxy method"\nmsgid "([^"]*)"\nmsgstr "([^"]*)"\n'
+    pattern = 'msgid "([^"]*)"\nmsgstr "([^"]*)"\n'
     proxy_methods = re.findall(pattern, fdata, flags=re.MULTILINE)
     proxy_methods = dict(proxy_methods)
     translations = {}
-    for k in ['None', 'Manual', 'Automatic']:
+    for k in ['Off', 'Manual', 'Automatic']:
```

Comment:
A more elegant solution would be to extract those terms from `po.template`.

Reply:
True, would be better to read the template for this purpose. Would you like to make that change? Also happy to leave it as is given it's not often we have to change these terms.

Reply:
If you don't mind, I'll leave the python magic for you :).
I guess since this pattern no longer just matches the proxy context, all translations are actually now inside `proxy_methods`, thus all translation tokens from the template can be found in this dictionary. So I think there is room for more optimizations.